recipes/prophyle/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```

Comment:
@bgruening Hi Björn, thank you for your comment. ProPhyle is still in a testing phase and does not have any Github releases, yet. This is why the source section is currently not used. Once a stable version is ready, we plan to create a standard Github release and use the source section of the receipt properly.

However, we might use the source section already now – with the PyPI releases (https://pypi.python.org/pypi?:action=display&name=prophyle).  Do you know how to derive a PyPI package address from the version (to be able to update the receipt just by incrementing a single Jinja variable)?